src/components/common.tsx

```diff
@@ -64,8 +64,10 @@ export const PanelContent = styled.div`
 export const PanelHorizontalLine = styled.div<{
   top?: number;
   bottom?: number;
+  left?: number;
+  right?: number;
 }>`
-  margin: ${({ top = 24, bottom = 16 }) => `${top}px 16px ${bottom}px 16px`};
+  margin: ${({ top = 24, bottom = 16, left = 16, right = 16 }) => `${top}px ${right}px ${bottom}px ${left}px`};
```

Comment:
This looks weird. Don't it break other places this component is used?

Reply:
It definitely doesn't break anything.
Before the change: "16px" was always used. Neither left nor right were used in params.
After the change: If left/right are omitted (as previously) 16px is set.